From adbb83f8dfd4dff638a57ca220634f9dfd1f565b Mon Sep 17 00:00:00 2001
From: Steve Muckle <smuckle@google.com>
Date: Mon, 9 Oct 2017 15:06:34 -0700
Subject: [PATCH 07/24] selftests/exec: use /system/bin/sh

The Android shell is located at /system/bin/sh.

(cherry picked from commit 860cea0bba7f78cf4cbb78251849e6ef159e6914)
Bug: 67016227
Test: run vts-kernel -m VtsKernelLinuxKselftestStaging
Signed-off-by: Steve Muckle <smuckle@google.com>
---
 tools/testing/selftests/exec/execveat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/exec/execveat.c b/tools/testing/selftests/exec/execveat.c
index 67bf7254a48f0..ecc974e75a9ba 100644
--- a/tools/testing/selftests/exec/execveat.c
+++ b/tools/testing/selftests/exec/execveat.c
@@ -379,7 +379,7 @@ static int run_tests(void)
 static void prerequisites(void)
 {
 	int fd;
-	const char *script = "#!/bin/sh\nexit $*\n";
+	const char *script = "#!/system/bin/sh\nexit $*\n";
 
 	/* Create ephemeral copies of files */
 	exe_cp("execveat", "execveat.ephemeral");
-- 
2.36.0.550.gb090851708-goog

